src/custom-element.js

```diff
@@ -528,6 +535,20 @@ export function createAmpElementProto(win, name, implementationClass) {
       this.style.width = assertLength(this.sizeList_.select(
           this.ownerDocument.defaultView));
     }
+    // Heights.
+    if (this.heightsList_ === undefined) {
+      const heightsList_ = this.getAttribute('heights');
+      this.heightsList_ = heightsList_ ?
+          parseSizeList(heightsList_, true) : null;
+    }
+
+    if (this.heightsList_ && this.layout_ ===
+        Layout.RESPONSIVE && this.sizerElement_) {
+      if (this.sizerElement_) {
+        this.sizerElement_.style.paddingTop = assertLength(
```

Comment:
We don't need an `assertLength` here since parser already verified that. If you don't mind could you please remove it above as well in `this.style.width = assertLength(this.sizeList_.select(`?